### PR TITLE
Bug 1340532, added clarifying details to Limiting Number of Self-Prov…

### PR DESCRIPTION
**Pull request by ahardin-rh**
…isioned Projects Per User

https://bugzilla.redhat.com/show_bug.cgi?id=1340532

### CODE REVIEW

**Comment by ahardin-rh** [comment]
@rjhowe @csrwng PTAL

**Comment by csrwng** [comment]
@deads ptal

**Comment by deads2k** [comment]
lgtm

**Comment by ahardin-rh** [comment]
@bfallonf @tnguyen-rh @tpoitras please peer review. Thanks!

**Comment by tpoitras** [comment]
@ahardin-rh LGTM! :cactus:

File: architecture/additional_concepts/admission_controllers.adoc [hunk 1]
```diff
@@ -22,6 +22,18 @@ system configured defaults. In addition, admission control plug-ins may modify
 related resources as part of request processing to do things such as
 incrementing quota usage.
 
+[WARNING]
+====
+The {product-title} master has a default list of plug-ins that are enabled by
+default for each type of resource (Kubernetes and {product-title}). These are
+required for the proper functioning of the master. It is not recommended to
```
Comment (bfallonf) [inline architecture/additional_concepts/admission_controllers.adoc]: "It is not recommended..." is expletive. Maybe:

"Modifying these lists is not recommended..."

**Comment by bfallonf** [comment]
@ahardin-rh One nit, otherwise looks good.

**Comment by ahardin-rh** [comment]
@csrwng Updated per your comments in the BZ. PTAL. Thanks!

File: admin_guide/manage_users.adoc [hunk 2]
```diff
@@ -66,6 +66,33 @@ authentication provider after starting to use {product-title}, the user names
...
+For example, if the user name is *theuser* and the label is *level=gold*:
+
+----
+$ oc label user/theuser level=gold
+----
+
+To remove the label:
+
+----
+$ oc label user/<user_name> -<label_name>
```
Comment (csrwng) [inline admin_guide/manage_users.adoc]: oc label user/<user_name> <label_name>-

**Comment by csrwng** [comment]
@ahardin-rh just one small change, otherwise LGTM

**Comment by ahardin-rh** [comment]
@csrwng Fixed. Thanks!

**Comment by ahardin-rh** [comment]
[rev_history]
|xref:../admin_guide/manage_users.adoc#admin-guide-manage-users[Managing Users]
|Added a new xref:../admin_guide/manage_users.adoc#managing-users-managing-user-and-group-labels[Managing User and Group Labels] section.
%
|xref:../admin_guide/managing_projects.adoc#admin-guide-managing-projects[Managing Projects]
|Added a Note box in the xref:../admin_guide/managing_projects.adoc#limit-projects-per-user[Limiting Number of Self-Provisioned Projects Per User] section with a pointer to the new xref:../admin_guide/manage_users.adoc#managing-users-managing-user-and-group-labels[Managing User and Group Labels] section.
%
|xref:../architecture/additional_concepts/admission_controllers.adoc#architecture-additional-concepts-admission-controllers[Additional Concepts -> Admission Controllers]
|Added a Warning box indicating that {product-title} master has a default list of plug-ins that are enabled by default for each type of resource.
%

**Comment by vikram-redhat** [comment]
@adellape @ahardin-rh - we may have an issue here. IIUC, Admissions Control is a 3.2 thing so the changes for that file and the link in the Rev History only applied to the 3.2 branch. 

When publishing to the portal, this causes an issue as this file doesn't exist in the 3.1 branch and the links fail.

I am not sure what the solution is here, but let's discuss in our next meeting.

**Comment by adellape** [comment]
@vikram-redhat @ahardin-rh Yeah, it was just mislabeled. The solution would have been to just drop the 3.1 label and not pick it for the 3.1 release. Looks like it has been picked though:

https://github.com/openshift/openshift-docs/commit/aae47fa7d306937d85fc2a41c6b760c59c4e8afa

At this point, we could do a revert on enterprise-3.1 (and clean up the revhistory).

**Comment by vikram-redhat** [comment]
@adellape Ok. I nominate @ahardin-rh for reverting as she labelled it in the first place. :) :dash:

**Comment by ahardin-rh** [comment]
@vikram-redhat @adellape FYI: This was resolved before yesterday's weekly release:
https://github.com/openshift/openshift-docs/pull/2588 :beers:

**Pull request merged**